woocommerce/woocommerce
# CYS: Remove iframe animation

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:











| Before | After |
|--------|--------|
| <video src="https://github.com/woocommerce/woocommerce/assets/4463174/5be49a94-02bf-4c38-a2a5-3fa3c23a6937" /> | <video src="https://github.com/woocommerce/woocommerce/assets/4463174/c326909d-bca6-4882-b6ce-b699d53a1f1b" /> | 

This PR removes the hover animation effect. The reasons for this change are:
- UX Improvement: The animation does not provide any significant benefit to the user experience.
- Performance Enhancement: The animation appears to cause some rerendering of elements within the iframe, potentially impacting performance.

Also, this PR removes the shadow around the iframe.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
7. Hover the site preview and ensure that any animation is triggered.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Remove iframe animation

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
